pyquil/_parser/grammar.lark

```diff
@@ -53,7 +54,8 @@ def_gate : "DEFGATE" name [ variables ] ":" matrix -> def_gate_matrix
 pauli_terms : ( _NEWLINE_TAB pauli_term )*
 pauli_term : name "(" expression ")" qubit_variables
 
-def_circuit : "DEFCIRCUIT" name [ variables ] [ qubit_designators ] ":" indented_instrs
```

Comment:
Is it understood why this wasn't working as expected? Is this a bug in `lark`?

Reply:
It was a bug in my testing. It actually was supported, and the issue was elsewhere